spec/roles.yml

```diff
@@ -225,8 +245,6 @@ components:
             $ref: 'openapi.yml#/components/responses/InadequatePrivileges'
           "404":
             $ref: 'openapi.yml#/components/responses/ResourceNotFound'
-          "422":
-            $ref: 'openapi.yml#/components/responses/UnprocessableEntity'
```

Comment:
There is a test case that confirms this behavior.

Reply:
Yeah, just added it in. The reason I removed it is because the Conjur documentation doesn't list it as valid. Conjur issue will probably arise from this.